src/witan/models/dem/ccm/inputs.clj

```diff
@@ -44,6 +44,12 @@
    :witan/key :future-mortality-trend-assumption
    :witan/schema s/NationalTrendsSchema})
 
+(definput in-future-fert-trend-1-0-0
```

Comment:
https://github.com/MastodonC/witan.models.demography/blob/feature/implement-fert-and-mort-variants/datasets/default_datasets/fertility/future_fertility_trend_assumption.csv